src/battle/area/iwa/actor/buzzar.c

```diff
@@ -1126,19 +1126,19 @@ EvtScript N(8021F030) = {
     EVT_END
 };
 
-EvtScript N(8021F7EC) = {
-    EVT_EXEC_WAIT(N(8021F824))
+EvtScript N(beginFlyToHome) = {
```

Comment:
Generally when we have these wrapper scripts that just execute another script, the outer one is names EVS_Name and the inner one is EVS_Name_Impl. So this would be EVS_FlyToHome and the script its executing would be EVS_FlyToHome_Impl.